src/app/services/UI_tools/alertService.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
```

Comment:
Nie wiem, jak wygląda w tym miejscu drzewko Gita, ale jestem przekonany, że ten plik widziałem już na masterze, i to w nowszej wersji?